src/main/resources/com/microsoft/jenkins/azuread/AzureAdMatrixAuthorizationStrategy/config.jelly

```diff
@@ -99,17 +99,19 @@ THE SOFTWARE.
         </j:forEach>
         <local:isEditable sid="${attrs.sid}" type="${attrs.type}">
           <td class="stop azure-ad-controls" style="text-align:left;">
-            <a href="#" class="selectall">
-              <img alt="${%Select all}" title="${%selectall(attrs.sid)}" src="${rootURL}/plugin/matrix-auth/images/select-all.svg" style='margin-right:0.2em' height="16" width="16"/>
-            </a>
-            <a href="#" class="unselectall">
-              <img alt="${%Unselect all}" title="${%unselectall(attrs.sid)}" src="${rootURL}/plugin/matrix-auth/images/unselect-all.svg" height="16" width="16"/>
-            </a>
-            <j:if test="${(attrs.sid != 'authenticated' or attrs.type != 'GROUP') and (attrs.sid != 'anonymous' or attrs.type != 'USER')}">
-              <a href="#" class="remove">
-                <l:icon alt="${%Remove user/group}" title="${%remove(attrs.sid)}" class="icon-stop icon-sm" />
+            <div class="mas-table__cell">

```

Comment:
this class is specific to Matrix-auth. Is it available in azure-ad?

Reply:
yeah it gets loaded on the page:
![image](https://github.com/jenkinsci/azure-ad-plugin/assets/21194782/0d34028d-d6eb-467b-9cf7-13e10cca1775)